server/base/src/main/java/org/apache/accumulo/server/master/balancer/RegexGroupBalancer.java

```diff
@@ -68,7 +69,8 @@ protected long getWaitTime() {
   @Override
   protected Function<KeyExtent,String> getPartitioner() {
 
-    Map<String,String> customProps = configuration.getTableConfiguration(tableId).getAllPropertiesWithPrefix(Property.TABLE_ARBITRARY_PROP_PREFIX);
+    Map<String,String> customProps = tabletBalancerEnvironment.getConfiguration().getTableConfiguration(tableId)
```

Comment:
Not everywhere. Most places are internals. I'm more concerned about the balancer, because this is a user-pluggable thing, and there's a good chance I'm going to change the TableConfiguration API significantly (actually, all of the AccumuloConfiguration classes).